webapps-2019.html

```diff
@@ -884,7 +887,188 @@ <h2>
           Incubator Community Group (WICG)</a> have been transferred to the
           WebApps Working Group.
         </p>
-      </section>
+	  </section>
+	  <section id="deliverablesfull">
+	  <h2>Detailed list of Deliverables</h2>
+	  <dl>
+	    <dt id=751 class='spec'><a href='https://www.w3.org/TR/FileAPI/' rel='versionof'>File API</a>
+	      <br>Latest publication: <a
+	      href='https://www.w3.org/TR/2018/WD-FileAPI-20181106/'>06 November
+	      2018</a></dt>
+	    <dd>Reference Draft:
```

Comment:
Aren't you missing the CfE at
  https://lists.w3.org/Archives/Member/member-cfe/2013Sep/0004.html
?

Reply:
I will seek confirmation, it is either a bug in the database or another rule that made that WD the reference draft, and not the FPWD.